lib/src/interfaces/Options.ts

```diff
@@ -919,6 +919,12 @@ export interface Options {
    * #### (Android specific)
    */
   modalTransitionStyle?: OptionsModalTransitionStyle;
+  /**
+   * In case of a "pageSheet" modalPresentationStyle, can you dismiss the modal by swiping
+   * 
+   * #### (iOS specific)
+   */
+  modalInPresentation?: boolean;
```

Comment:
Lets rename the property we expose through options to `dismissGesture`. `modalInPresentation` is coupled to iOS implementation and naming conventions.

Reply:
Ok renamed the option in the upcoming push! 👍

Reply:
I actually renamed it to disableDismissGesture because the dismiss gesture is enabled by default. It also directly maps to the API (you can turn it **off** by setting the modalInPresentation option)